### PR TITLE
Add optional device name seperate from alias

### LINKED ISSUE #48: Support a Friendly Name
Allow the name in Home Assistant for the device to contain spaces and be separate from the bluetooth alias and mqtt topic.
In Tasmota this is known as the friendly_name and it is separate and can be set independently. As far as I can tell there is currently no way to have a space in the name presented to Home Assistant other than to edit it in home assistant after the fact.

Some ideas:
use string_replace on alias and replace spaces. Then use the replacement for mqtt topic and bluetooth alias while using original alias as Name for discovery etc.

Add a new attribute called `friendly_name` to allow the setting of the home assistant name independently of the topic and bluetooth alias.

### PR DESCRIPTION
**Pull request by edestecd**
Closes #48

### CODE REVIEW

File: blerry/blerry.be [hunk 1]
```diff
@@ -360,6 +361,7 @@ class Blerry_Device
   var next_forced_publish
 
   def init(mac, config, blerry_inst)
+    self.name = config.find('name', config['alias'])
```
Comment (tony-fav) [inline blerry/blerry.be:364]: This line means "if name is not in the config, use the alias as the name", yeah? Can you adjust the comment you added to the readme to clarify?

Comment (edestecd) [reply blerry/blerry.be:364]: correct. Yea I can adjust the comment

Comment (edestecd) [reply blerry/blerry.be:364]: This should not change behavior for any existing users

**Comment by edestecd** [comment]
Yea Just tested the last change and think I'm done for now

**Comment by tony-fav** [comment]
Thanks for the contribution!

**Pull request merged**